.github/workflows/nodejs.yml

```diff
@@ -70,17 +70,7 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-
-      - name: Restore lerna cache
-        id: cache
-        uses: actions/cache@v2
-        with:
-          path: |
-            node_modules
-            */*/node_modules
-          key: c-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
-          restore-keys: |
-            c-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-
+          cache: 'yarn'
```

Comment:
We need better key here, otherwise cache was broken

Reply:
we need here `webpack-version` and `hashFiles`

Reply:
yep, I don't see it on the official docs, lemme see if something is possible

Reply:
The action computes a hash internally so we won't need hashFiles

Reply:
Can you provide line where it calculated?

Reply:
yep sorry should've done earlier - https://github.com/actions/setup-node/blob/e215578cb0305f14eb5d11939e0f4610ae5bf7e6/src/cache-restore.ts#L26